src/hotspot/share/prims/jvmti.xml

```diff
@@ -3078,8 +3078,8 @@ err = (*jvmti)-&gt;Deallocate(jvmti, stack_info);
       </parameters>
       <errors>
         <error id="JVMTI_ERROR_OPAQUE_FRAME">
-          The frame at <code>depth</code> is executing a
-          native method.
+          The implementation is unable to set a <eventlink id="FramePop"></eventlink> event for the frame
+          (e.g. the frame at <code>depth</code> is executing a native method).
```

Comment:
I don't think this sounds right. You don't "set" a FramePop event. You tell JVMTI to generate a FramePop event when this frame is popped. Maybe something like "A FramePop event cannot be generated for this frame".

Reply:
Chris, please, let me know if it is okay to keep this at is or you still want an update for the `NotifyFramePop`.

Reply:
I understand what you are saying about the comparison to SetBreakpoint and that SetFramePop would have been a better name, but I don't see what this with the description. I said "set a FramePop event" doesn't sound right. Neither would "set a Breakpoint event". Also, the analogy with breakpoints doesn't really work when you consider that "a breakpoint" is meaningful, but "a FrampPop" does not.

A breakpoint is a location in the code where a breakpoint event will be triggered when executed. That analogy doesn't really work with FramePop. We don't have a word similar to breakpoint that means "a frame marked for generating a FramePop event when popped".